src/main/kotlin/study/racingcar/Car.kt

```diff
@@ -0,0 +1,11 @@
+package study.racingcar
+
+class Car {
+
+    private var _distance: Int = 0
+    val distance: Int get() = _distance
```

Comment:
setter만 비공개해줄 수도 있습니다.

```suggestion
    val distance: Int = 0
        private set
```

Reply:
val 인데 값을 처음 지정후 다른 값으로 셋팅 할 수 없지 않나요

Reply:
앗! 맞습니다!
`val`이 아니라 `var`로 선언해야 합니다.
```kotlin
var distance: Int = 0
    private set
```